Dnn.CommunityForums/DnnCommunityForums.csproj

```diff
@@ -1012,6 +1012,8 @@
     <None Include="sql\07.00.02.SqlDataProvider" />
     <None Include="sql\07.00.00.SqlDataProvider" />
     <None Include="sql\07.00.02.SqlDataProvider" />
+    <None Include="sql\08.00.03.SqlDataProvider" />
```

Comment:
Should this version be here? Should we put that SQL into the 8.1 SQL?

Reply:
> Should this version be here? Should we put that SQL into the 8.1 SQL?

Please merge 636, 637, and 638 first ....
![image](https://github.com/DNNCommunity/Dnn.CommunityForums/assets/9553126/4210a93b-8d6f-4532-99d2-5ca17b77affc)